.github/workflows/maven-cd.yml

```diff
@@ -51,14 +51,17 @@ jobs:
       - name: Log should_release details
         id: log-should_release-details
         run: |
-          echo "================================"
-          echo "Release job filter details:"
-          echo "          VALIDATE_ONLY: ${VALIDATE_ONLY}"
-          echo "              CI_STATUS: ${CI_STATUS}"
-          echo "   INTERESTING_CATEGORY: ${INTERESTING_CATEGORY}"
-          echo "--------------------------------"
-          echo "=>       SHOULD_RELEASE: ${SHOULD_RELEASE}"
-          echo "================================"
+          echo "## Job Filter Summary"                                  >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`"                                                 >> $GITHUB_STEP_SUMMARY
+          echo "================================"                 | tee -a $GITHUB_STEP_SUMMARY
```

Comment:
BTW here-docs would be a bit more legible.

Reply:
@jglick Yes, thought about that as well afterwards - will file another PR if I have few minutes.